src/e2e-test/java/io/cdap/plugin/utils/PubSubClient.java

```diff
@@ -56,4 +101,92 @@ public static String getTopicCmekKey(String topicId) throws IOException {
     return getTopic(topicId).getKmsKeyName();
   }
 
+
+  public static void publishWithErrorHandlerExample(String projectId, String topicId, List<String> dataMessages)
```

Comment:
change the function name to something more relatable, try removing example from the method name

Reply:
publishMessagesWithPubSub